### DISCUSSION TITLE
Page groups should be linkable and collapsible

### DISCUSSION THREAD

**Discussion by ahcsco** [GitbookIO/community#138] [Feature Requests]
### What's your feature/idea?

We are creating a start page for our docs and we'd like that main topics are referrable to page groups.
Currently not possible in Gitbook to link to page group level, only to individual pages under the group. This hinders us from linking directly to a main topic of interest which is page group.

It'd be great to have this possible, like below. Page group should be 1) linkable with a standard url, and 2) clickable which should result in it getting expanded/collapsed.

<img width="317" alt="Screenshot 2023-03-10 at 09 45 24" src="https://user-images.githubusercontent.com/112613597/224268126-043b3cbf-8fed-4cbd-b893-863407f2ecb8.png">

### What's your desired outcome?

Be able to link to page groups and expand / collapse them.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by aikoii** [comment]
Related to this request 🙂: https://github.com/orgs/GitbookIO/discussions/15

**Comment by indexzeroxyz** [comment]
Defintely a must have for me too

**Comment by taszap** [comment]
Related to #15

**Comment by aikoii** [comment]
Hey folks! The page groups are now collapsible 👍 See https://github.com/orgs/GitbookIO/discussions/708#discussioncomment-16730009
